lib/registries/addon/drafts/draft/review/style.scss

```diff
@@ -0,0 +1,91 @@
+.ContentBackground {
```

Comment:
Rename files from
`lib/registries/addon/drafts/draft/review/style.scss`  to
`lib/registries/addon/drafts/draft/review/styles.scss`